client/src/main/scala/eventstore/akka/Settings.scala

```diff
@@ -60,12 +61,11 @@ final case class Settings(
 
 object Settings {
 
-  lazy val EsDefault = EsSettings()
-  lazy val Default   = Settings(ConfigFactory.load())
+  lazy val Default = Settings(ConfigFactory.load())
```

Comment:
in general I would ban all `ConfigFactory.load()` calls :)

Reply:
Agree :)